pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/JUnitTestsShouldIncludeAssert.xml

```diff
@@ -429,6 +429,23 @@ class Style {
   public void moveOutOfBoundsFrom() {
     doSomething();
   }
+}]]></code>
+    </test-code>
+    <test-code>
+        <description>#??? Treat AssertJ soft assertions as assert expressions</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Test;
+
+class FooTest {
+
+    @Test
+    void testFoo() {
+        var softly = new SoftAssertions();
```

Comment:
Could you please extend this test case to add a second method, which uses the "traditional" variable declaration?

```java
@Test
    void testFoo() {
        SoftAssertions softly = new SoftAssertions();
        ...
```